index.js

```diff
@@ -40,7 +53,8 @@ app.post("/login", (req, res) => {
         if(user) {
             let userdata = {
                 username: uid,
-                password: pd
+                password: pd,
+                role:user.role

```

Comment:
what is this role ?

Reply:
I will send user.json file ,upload this in your mongodb